app/templates/route-components/tabs.hbs

```diff
@@ -0,0 +1,121 @@
+<h1>Tabs</h1>
+<p>
+	Tabs are used for handling content in a more ordered way and also may be used to build complex navigation solutions.
+</p>
+
+<section>
+	<h2 id='text-only'>Tabs with text only</h2>
+	<p>
+		To display a set of tabs you need to place them inside <code>&lt;ul&gt;</code> tag with class <code class='format-code'>.wds-tabs-container</code>
+		Each <code>&lt;li&gt;</code> item (tab) needs to have <code class='format-code'>.wds-tab</code> class and some link <code>&lt;a&gt;</code> or plain text
+		<code>&lt;span&gt;</code> inside with class <code class='format-code'>.wds-tab-label</code>.
```

Comment:
What do you think about having these?
- `.wds-tabs`
- `.wds-tabs__tab`
- `.wds-tabs__tab-label`?

That would follow what we have in e.g. `/identity/global-footer`:
```
<footer class="wds-global-footer">
  <h2 class="wds-global-footer__header">	
    <svg class="wds-global-footer__header-logo">
```

There is no single tab without the tabs container, so all the elements together are basically a single component, right? The naming should reflect that.